ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -2586,3 +2586,39 @@ def test_paddle_instance_sgn(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# bmm
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="bmm",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=(3, 3, 3),
```

Comment:
This seems a little too specific. We should ideally be testing on a subset of the 3D space of inputs here. 
So the input shapes should be `(b, m, k)`  and `(b, k, n)` where b,k,n could be any integers. 
You could try creating a composite helper to achieve this. Or perhaps something similar to this already exists (not sure though aha 😅 )

Reply:
I checked ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_linalg.py and in this file test bmm also tested that way. Check it please.

Reply:
Yeah, I checked it, and it seems like a slip up most probably from the PR reviewer. Ideally, we should be testing for all corner cases rather than testing for only a subset just to get the tests passing.